### DISCUSSION TITLE
My License Not there

### DISCUSSION THREAD

**Discussion by aashirwad0209** [aclap-dev/video-downloadhelper#446] [License & Purchase requests]
### What's your issue?

I purchase the license on Tue, 1 Sept 2020 (My email is aashirwad1@gmail.com

Order Details as mentioned below, but its not working now

Chrome Web Store
1 September 2020, 01:28:06 GMT-7
Item	Quantity	Price
Lifetime Premium VDH	1	₹914.00
Integrated GST (18%)   ₹164.52

Total₹1,078.52
Payment method
Visa  • • • • 5884
Order number
CWS.9315-3199-9669-53423

### The 10 first digits (only 10) of your license number or email address

aashirwad1@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by aashirwad0209** [comment]
REMINDER

**Reply by jcfrog** [reply]
hi,
I sent you an email.
